user/models.go

```diff
@@ -64,6 +64,7 @@ type User struct {
 	Phone             string                 `bson:"phone" validate:"required" json:"phone"`
 	Settings          *UserSettings          `bson:"settings" json:"settings"`
 	Timezone          string                 `bson:"time_zone" json:"time_zone"`
+	Role              string                 `bson:"role" json:"role"`
```

Comment:
Let's convert it to array/dictionary, and declare the array of roles in auth model

Reply:
Like I said earlier and have told Jb, their is no need to assign multipe roles to a user